pkg/route/api/v1/conversion.go

```diff
@@ -13,6 +13,9 @@ func init() {
 			obj.To.Kind = "Service"
 		},
 		func(obj *TLSConfig) {
+			if len(obj.Termination) == 0 && len(obj.DestinationCACertificate) == 0 {
+				obj.Termination = TLSTerminationEdge
```

Comment:
> @deads2k how about initializing to Re-encrypt when there is a DestinationCACertificate?

I'm less certain about that.  @pweil-  may know.  Seems like re-encryption would require addition keys.